### ISSUE TITLE
Return simple 200 on GET to /

### ISSUE THREAD

**Issue by milescrabill** [mozilla/speech-proxy#25]
Once we're in production and the URL is in the Firefox client courtesy of the webextension, it might receive userfacing traffic

**Comment by andrenatal** [comment]
This is the error currently happening. 

```
TypeError: Invalid data, chunk must be a string or buffer, not object
    at Socket.write (net.js:696:11)
    at /home/andre/voicefill/speech-proxy/server.js:140:17
    at Layer.handle [as handle_request] (/home/andre/voicefill/speech-proxy/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/andre/voicefill/speech-proxy/node_modules/express/lib/router/index.js:317:13)
    at /home/andre/voicefill/speech-proxy/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/home/andre/voicefill/speech-proxy/node_modules/express/lib/router/index.js:335:12)
    at next (/home/andre/voicefill/speech-proxy/node_modules/express/lib/router/index.js:275:10)
    at rawParser (/home/andre/voicefill/speech-proxy/node_modules/body-parser/lib/types/raw.js:67:7)
    at Layer.handle [as handle_request] (/home/andre/voicefill/speech-proxy/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/andre/voicefill/speech-proxy/node_modules/express/lib/router/index.js:317:13)
```

**Comment by andrenatal** [comment]
Fixed by #40

**Comment by umadevisk89** [comment]
I see  "Invalid data, chunk must be a string or buffer, not object
    at Socket.write (net.js:696:11)" error on connecting to ldap using ldapjs module
